### PR TITLE
Adjust jn:null, jdm:keys, newlines at end of tests

### PR DESCRIPTION
**Pull request by riyafa**
Based on post commit comments

### CODE REVIEW

File: vxquery-core/src/main/java/org/apache/vxquery/functions/builtin-functions.xml [hunk 1]
```diff
@@ -1158,7 +1158,7 @@
 
     <!-- jn:null() as js:null -->
     <function name="jn:null">
-        <return type="xs:string*"/>
+        <return type="js:null*"/>
```
Comment (westmann) [inline vxquery-core/src/main/java/org/apache/vxquery/functions/builtin-functions.xml]: Should just be "js:null" as it always returns exactly one null value.

**Comment by westmann** [comment]
Looks good to me.

File: vxquery-core/src/main/java/org/apache/vxquery/functions/builtin-operators.xml [hunk 2]
```diff
@@ -655,9 +655,10 @@
         <runtime type="scalar" class="org.apache.vxquery.runtime.functions.json.ValueScalarEvaluatorFactory"/>
     </operator>
 
+    jdm:keys($o as object()) as xs:string*
```
Comment (prestoncarman) [inline vxquery-core/src/main/java/org/apache/vxquery/functions/builtin-operators.xml]: This should be a xml comment.

**Comment by prestoncarman** [comment]
Merged squashed version of this change.

**Pull request closed without merging**